### PR TITLE
Adding android screenshots and texts

### PR DESCRIPTION
**Pull request by tclaus**

### CODE REVIEW

File: metadata/fr/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Un client natif <a href='https://diasporafoundation.org'>diaspora*</a> alimenté par Flutter.
```
Comment (Flaburgan) [inline metadata/fr/full_description.txt]: Un client natif <a href='https://diasporafoundation.org'>diaspora*</a> *fait avec* Flutter.

File: metadata/fr/full_description.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+Il s'agit d'une version préliminaire. Vous aurez besoin d'un pod sur la dernière branche de développement. Les contributions sont les bienvenues, merci d'ouvrir un dossier si vous travaillez sur quelque chose de plus important afin d'éviter la duplication des efforts :)
```
Comment (Flaburgan) [inline metadata/fr/full_description.txt]: Vous aurez besoin d'un pod utilisant la version de développement pour l'utiliser. Les contributions sont les bienvenues, merci d'ouvrir une issue github si vous souhaitez contribuer afin d'éviter la duplication des efforts :)

**Review by Flaburgan — CHANGES_REQUESTED** [review]
Hey, thanks for this! You can also use my french screenshots if the store allows to have several languages.

**Comment by tclaus** [comment]
Hm - all the images I took were missing in the commits - I' add them in  the next commit.

**Comment by IzzySoft** [comment]
And any specific reason you used `de-DE` for the German locale, and not just `de`? Is it specific to the German spoken in Germany and not fitting our Austrian friends?

One more note: You seem to have copied the icon to all locales. That is not necessary. If it's missing in one locale, clients fall back to `en-US` and look for it there. Same for `title.txt`. You only need to provide for other locales what shall *not* be taken from `en-US`.

File: metadata/de/short_description.txt
Comment (IzzySoft) [inline metadata/de/short_description.txt:1]: Is it really important to those potentially using the app that it is "based on Flutter"? What does it tell people who are "not techy" (apart from maybe saying "that's why the APKs are bigger")? Does anyone choose apps based on their development framework?

Remember "short description" is what shows up in lists, so this is where the "first decision" is made: "Is this interesting to me?" It's limited to 80 chars, so this space is better used for "eye catchers". What is it that makes the app interesting to you? Terms like "easy to use" or "intuitive" come to mind. I'm not on Diaspora and hence not familiar with the app, so I cannot suggest what *should* be here instead.

Same for short descs of the other locales of course.

**Review by IzzySoft — APPROVED** [review]
LGTM, suggested changes are "nice to have" but not urgent :wink:

File: metadata/de/short_description.txt
Comment (tclaus) [reply metadata/de/short_description.txt:1]: Totally agree. "Flutter" in an app description is kind of techy - I have changed it to something more with a social media relation.

**Pull request merged**